caikit/core/data_model/base.py

```diff
@@ -589,8 +588,8 @@ def from_proto(cls, proto):
                 A protocol buffer to serialize from.
 
         Returns:
-            protobufs
-                A DataBase object.
+			protobufs: A DataBase object.
```

Comment:
hah, this is another funky case of the named return problem. `protobufs` is not a type but `DataBase` is...

Dunno how you'd catch this case, we'll probably just need to come clean these up to read more like
```
Returns:
    DataBase: An instance of the data model class
```